CollectionViewDemo/IconDetailViewController.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  IconDetailViewController.swift
+//  CollectionViewDemo
+//
+//  Created by Laurentiu Ile on 19/01/2021.
+//
+
+import UIKit
+import IconDataKit
+
+class IconDetailViewController: UIViewController {
+    
+    @IBOutlet var nameLabel: UILabel! {
+        didSet {
+            nameLabel.text = icon?.name
+        }
+    }
+    @IBOutlet var descriptionLabel: UILabel! {
```

Comment:
please leave a space between the fields if they have didSet. it is easier to read